docs/pbc/all/warehouse-management-system/202311.0/unified-commerce/fulfillment-app-feature-overview.md

```diff
@@ -0,0 +1,112 @@
+
+In the Back Office, to create a warehouse user, you need to select a respective option when [creating](/docs/pbc/all/user-management/202311.0/base-shop/manage-in-the-back-office/manage-users/create-users.html) or [editing](/docs/pbc/all/user-management/202311.0/base-shop/manage-in-the-back-office/manage-users/edit-users.html) a regular Back Office user.
+
+To give a warehouse user access to fulfilling orders in a warehouse, you need to assign the warehouse to them. For instructions on how to do that, see [Assign and deassign warehouses from warehouse users](/docs/pbc/all/warehouse-management-system/202311.0/unified-commerce/assign-and-deassign-warehouses-from-warehouse-users.html).
+
+## Fulfillment App
+
+Fulfillment App is an app used by warehouse users to fulfill orders. When a customer places an order, it is assigned to a warehouse that can fulfill the order. In the assigned warehouse, a warehouse user uses the Fulfillment App to gather and ship the order items.
+
+![screenshot with two picklists]
+
+## Assigning a warehouse
+
+By default, when a customer orders several items of the same SKU, the requested item’s stock is checked in all of the store's warehouses. The warehouses are sorted in the descending order of how much stock of the item they are holding. The warehouse holding the biggest stock of the item is assigned to the order. The warehouse holding the never-out-of-stock item quantity is considered to be holding the biggest stock.
+
+
+In some cases, the ordered quantity might not be available in the warehouse holding the biggest stock of the item. Then, the warehouse fulfills the order partially. The remaining quantity is fulfilled by the the warehouse holding the next biggest stock of the item. For example, the customer orders 100 tablets. The store's warehouses hold the following stock:
+
+| WAREHOUSE | STOCK |
+|-|-|
+| 2 | 65 |
+| 3 | 40 |
+| 1 | 13 |
+
+In this case, warehouse 2 provides 65 tables by emptying out its stock completely. Warehouse 3 provides the remaining 35 tablets with 5 tablets left in stock.
+
+
+{% info_block warningBox "Item reservation" %}
+Warehouse allocation shouldn't be confused with the reservation process, meaning items are not reserved when they are allocated to a warehouse. The warehouse management system is responsible for warehouse reservation. The default strategy is designed to show how warehouse allocation can be implemented and used in the fulfillment process.
+{% endinfo_block %}
+
+
+## Warehouse picklists
+
+A *warehouse picklist* is a document available in the Fulfillment App that contains a list of items to be picked and shipped to fulfill and order. It includes the following information:
+* Product name
+* Product image
+* Quantity to be picked
+
+Based on a picklist, the warehouse user gathers and prepares the items for shipping.
+
+The information within will also allow you to quickly restock inventory as required.
```

Comment:
Could you clarify this point?

Reply:
We do not have any feature for re-stocking in particular. We can remove this line.